_posts/2020-01-26-newpipe-0.18.2-released.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "NewPipe 0.18.2 released: Hotfix for the YouTube Extractor"
+short: "NewPipe 0.18.2 released"
+date: 2020-01-26 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+8 days after the last release and the next one is already out: A hotifx for the Extractor! A change on YouTubes side made it necessary.
```

Comment:
```suggestion
8 days after the last release and the next one is already out: A hotfix for the Extractor! A change on YouTube's side made it necessary.
```

Reply:
@Poolitzer hasn't allowed us to edit their branch. Took me 2 minutes to figure out why this is shown:

![screenshot_2020-01-26_12-46-01](https://user-images.githubusercontent.com/4503202/73134678-d2131180-4039-11ea-8fb6-6a4c88a1a121.png)

Please commit this last change and we can merge and publish your post.

Reply:
But I ticked the box :(

Reply:
@TheAssassin I'll merge and push a commit to master fixing the typos.

Reply:
Fixed them already

Reply:
Oh. GitHub was a bit slow and didn't show me the new commit.
Thanks @Poolitzer!

Reply:
You are welcome